Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -192,6 +192,38 @@ Command keywords and values
          Apart from this, ordinary permissions apply as if the user wants to
          make a copy of the record on the same page.
 
+         The :code:`localize` DataHandler command should be used when translating records in "Connected Mode"
+         (strict translation of records from the default language).
+         This command is used when selecting the "Translate" strategy in the content elements translation wizard.
+
+
+ - :Command:
+         copyToLanguage
+   :Type:
+         integer
+   :Value:
+         It behaves like 'localize' command (both record and child records are copied to given language),
```

Comment:
A last syntax highlight for `localize` here and we're good to go from my POV.

@marble I'd do the syntax highlights for the whole file in a separate change.